Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,33 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "gcc" # for gfortran
+
+  def install
+    args = std_cmake_args
+    args << "-DCMAKE_INSTALL_PREFIX=/usr/local"
+    args << "-DCMAKE_INSTALL_LIBEXECDIR=/usr/local/lib"
+    system "cmake", ".", *args
+    system "make"
+    system "make", "docs"
+    system "make", "DESTDIR=#{prefix}", "install"
```

Comment:
This is really a hack. Add a short comment to explain why we do this.

Reply:
The kim-api-v2 package supports "plugins" called models and model-drivers  (actually its main purpose is really to be a plugin manager) and these plugins are stored in the "#lib/#{name}/models/" and "#lib/#{name}/model-drivers" directories.  I'm also creating a second package that will install a collection of 200 such plugins.  If I don't use this hack, that package tries to install its files in /usr/local/Cellar/kim-api-v2/2.0.0/lib/kim-api-v2/model-drivers/ and fails with a "cannot create directory" error.  I guess this could be dealt with by adding another CMAKE option to the package.

Any other suggestions for how to handle this better?

Reply:
I can't find an alternative solution...  Added a comment